docs/Versioning.md

```diff
@@ -114,18 +139,17 @@ Model authors and applications/systems MAY elect to ignore the model versioning
 
     * Breaking changes to the semantics of an input or output (e.g., changing the required contents of an input tensor from a color image to a black and white image).
     * Changing the declared type of an input or output to an incompatible type (e.g., `tensor(int)->tensor(string)`).
-    * Adding a new input for which there is no meaningful or specified default value. For graph inputs, those values are provided by a same-named value in GraphProto. initializer.
```

Comment:
“ For graph inputs, those values are provided by a same-named value in GraphProto. initializer.", it's better to keep this. This is telling "how to specify the default value of a graph input".

Reply:
that's also specified here, right?
https://github.com/edgchen1/onnx/blame/edgchen1/ir_doc_fixes/docs/IR.md#L159

would it be better to specify it once? that information also seemed out of place here.

Reply:
Well, no harm in reminding the reader that the default value is specified by the initializer. Most people are familiar with the term "initializer", but may not immediately correlate that to a "default value". How about adding back "Recall that default-values for inputs are specified in the initializer list."

Reply:
I agree to add some words back for more clarification.